platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -232,6 +245,21 @@ class EditorEditText : AppCompatEditText {
         }
     }
 
+    /**
+     * Sets up the [HtmlConverter] used to translate HTML to Spanned text.
+     * @param styleConfig The styles to use for the generated spans.
+     * @param mentionDisplayHandler Used to decide how to display any mentions found in the HTML text.
+     */
+    fun setupHtmlConverter(styleConfig: StyleConfig, mentionDisplayHandler: MentionDisplayHandler?) {
```

Comment:
The `HtmlConverter` is an implementation detail that could confuse users who are simply trying to change the appearance of the editor. Can we rename the public API to something a bit more friendly like `setStyle`?

Reply:
Sounds good, `setStyle/updateStyle` works for me.